Flutter/DateRangePicker/customizations.md

```diff
@@ -169,6 +178,12 @@ Widget build(BuildContext context) {
 
 ![Year cell customization Date Range Picker](images/customizations/yearcell_customization.png)
 
+>**NOTE**
+* Priority order
```

Comment:
change like above comment

Reply:
I have checked and changed it like above comment. I have updated in my latest commit.